tests/TestSuite/Mutant/ProcessTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ *
+ * @category   Humbug
+ * @package    Humbug
+ * @copyright  Copyright (c) 2015 Pádraic Brady (http://blog.astrumfutura.com)
+ * @license    https://github.com/padraic/humbug/blob/master/LICENSE New BSD License
+ * @author     Thibaud Fabre
+ */
+namespace Humbug\Test\TestSuite\Mutant;
+
+use Humbug\TestSuite\Mutant\Process;
+use Humbug\TestSuite\Mutant\Result;
+
+class ProcessTest extends \PHPUnit_Framework_TestCase
+{
+    public function testProperties()
+    {
+        $adapter = $this->prophesize('Humbug\Adapter\AdapterAbstract');
+        $adapter->ok("")->willReturn(true);
```

Comment:
I know it`s a double but this `$adapter->ok("")->willReturn(true)` prompts me that this is proper `adapter->ok()` method behavior.
Consider replacing it with `$adapter->ok(Argument::type('string'))->willReturn(true)`.